util/package.go

```diff
@@ -97,6 +98,11 @@ type Requirement struct {
 	Kibana ProductRequirement `config:"kibana" json:"kibana,omitempty" yaml:"kibana"`
 }
 
+type Conditions struct {
+	KibanaVersion string `config:"kibana.version,omitempty" json:"kibana.version,omitempty" yaml:"kibana.version,omitempty"`
+	kibanaVersion *semver.Constraints
```

Comment:
I suggest to rename this field, the difference is in capitals. How about `kibanaConstraint` or `kibanaSemver`?

Reply:
Renamed to `kibanaConstraint`.